docs/README.md

```diff
@@ -14,6 +14,7 @@ The following JSON is the default configuration for the module and can be overri
 		"altis": {
 			"modules": {
 				"seo": {
+					"development-mode": true,
```

Comment:
These aren't strictly alphabetical - I think `enabled` should really come first as it's a special option. Maybe `debug` might be a terser option? The `WPSEO_DEBUG` constant can also control this behaviour.